.vscode/settings.json

```diff
@@ -4,16 +4,24 @@
     "**/.yarn": true,
   },
   "eslint.nodePath": ".yarn/sdks",
-  "eslint.execArgv": ["--max_old_space_size=16384"],
-  "eslint.workingDirectories": [ "./packages", "./generators" ],
+  "eslint.execArgv": [
+    "--max_old_space_size=16384"
+  ],
+  "eslint.workingDirectories": [
+    "./packages",
+    "./generators"
+  ],
   "prettier.prettierPath": "node_modules/prettier/index.js",
   "typescript.tsdk": "node_modules/typescript/lib",
   "typescript.enablePromptUseWorkspaceTsdk": true,
-  "editor.formatOnSave": true,
-  "editor.codeActionsOnSave": ["source.fixAll", "source.sortMembers"],
+  "editor.formatOnSave": false,
```

Comment:
Was this intentional? It might be better to put this in your user settings that aren't committed to the repository.